pkg/helmx/chartify.go

```diff
@@ -33,6 +33,9 @@ type ChartifyOpts struct {
 	// TillerNamespace is the namespace Tiller or Helm v3 creates "release" objects(configmaps or secrets depending on the storage backend chosen)
 	TillerNamespace string
 
+	// EnableKustomizAlphaPlugins will add the `--enable_alpha_goplugins_accept_panic_risk` flag when running `kustomize build`
```

Comment:
The code comment mentions `--enable_alpha_goplugins_accept_panic_risk`. Should it be `--enable_alpha_plugins`?

Reply:
Ah yes, I forgot to rename here.
The flag was called `--enable_alpha_goplugins_accept_panic_risk` for a long time, and got renamed recently.